File: 0217-contains-duplicate/0217-contains-duplicate.cpp
class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
        sort(nums.begin(), nums.end());
        int j=1;
        bool flag = false;
        for(int i = 0; i < nums.size(); i++){
            if(j < nums.size()){
                if(nums[i] == nums[j]){
                    flag = true;
                    break;
                }
                else{
                    j += 1;
                }
            }
        }
        if(flag == true){
            return true;
        }
        else{
            return false;
        }
        
    }
};

File: 0217-contains-duplicate/0217-contains-duplicate.py
class Solution:
    def containsDuplicate(self, nums: List[int]) -> bool:
        j=1
        flag= False
        nums.sort()
        for i in range(len(nums)):
            if j < len(nums):
                if nums[i] == nums[j]:
                    flag = True
                    break
                else:
                    j += 1
        if flag:
            return True
        else:
            return False
         
        